src/icons/illustrations/UploadIllustration.js

```diff
@@ -0,0 +1,161 @@
+// @flow
+import * as React from 'react';
+import uniqueId from 'lodash/uniqueId';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const ICON_CLASS = 'upload-illustration';
+
+class UploadIllustration extends React.PureComponent<Props> {
+    static defaultProps = {
+        className: '',
+        height: 141,
+        width: 140,
+    };
+
+    idPrefix = `${uniqueId(ICON_CLASS)}-`;
+
+    render() {
+        const { className, height, title, width } = this.props;
+        return (
+            <AccessibleSVG
+                className={`${ICON_CLASS} ${className}`}
+                height={height}
+                title={title}
+                viewBox="0 0 141 140"
+                width={width}
+            >
+                <g transform="translate(0 9)" fill="none" fillRule="evenodd" xmlns="http://www.w3.org/2000/svg">
```

Comment:
SVG can be cleaned up quite a bit, to get rid of transforms and extra `<g>`.
Colors in the svg should be replaced with bdl color variables if there is a use case to override it.
See other new icons as example.